### ISSUE TITLE
Include an option to turn-off the indexing for public documentation

### ISSUE THREAD

**Issue by Ramya-From-Postman** [a85/POSTMan-Chrome-Extension#6826]
Hi Team,

Currently, public documentation pages are always indexed. Also, have the ability to add them in the API Network or as a template that is searchable by the community. However, it will be good if there an option included for turning off the indexing so that only the people who have the link can access the document.

**Comment by nikuscs** [comment]
This is simple as adding a no-index header.

**Comment by Raja-Simha** [comment]
@igerslike I would like to understand why you want to turn off indexing? Do you work with partner APIs and do you want the share those APIs only with your partners?

**Comment by nikuscs** [comment]
Exactly. We want to keep the api docs undiscoverable without any chance of being indexed.

**Comment by ramji367** [comment]
@nikuscs @Raja-Simha You can do this now using Partner Workspaces, available with Postman Professional and Enterprise plans. To share a Postman collection with embedded API documentation:

1. Create a [partner workspace](https://learning.postman.com/docs/collaborating-in-postman/using-workspaces/partner-workspaces/)
2. Add your collections to the workspace. Optionally you can also add Environments with pre-set values if you want partners to easily test your APIs
3. Hit the Invite button and select Partners. Add the email address of your partner and invite them to the workspace

You can also enable [Multi-partner mode](https://learning.postman.com/docs/collaborating-in-postman/using-workspaces/multipartner-workspaces/) if you want to invite partners from multiple organizations to the workspace.

**Comment by nikuscs** [comment]
Noice, it only took 5 years!
